stock_account_operating_unit/README.rst

```diff
@@ -26,7 +26,7 @@ be self-balanced, and then indicate a self-balancing clearing account.
 * Go to *Settings / Companies / Configuration* and:
    ** Set the "Operating Units are self-balanced" checkbox
    ** Set the "Inter-OU Clearing"  account in "Inter-operating unit clearing
-      account" field.
+   account" field.
```

Comment:
This was fine before the change. <a href="http://docutils.sourceforge.net/docs/user/rst/quickref.html#bullet-lists"> Continuing text must be aligned after the bullet and whitespace. </a>